cypress/e2e/WebInterface/Test Cases/TestCaseImport/TestCaseImportviaFile.cy.ts

```diff
@@ -0,0 +1,203 @@
+        //navigate to the main measures page
+        cy.get(Header.measures).click()
+
+        //click on created measure
+        MeasuresPage.clickEditforCreatedMeasure()
+
+        //click on the test case tab
+        cy.get(EditMeasurePage.testCasesTab).click()
+
+        //click on the import test case button
+        cy.get(TestCasesPage.importTestCasesBtn).click()
+
+        //select file
+        cy.get(TestCasesPage.filAttachDropBox).attachFile(largeBatchTestCaseFile)
+
+        //click on the 'Import' button on the modal window
+        TestCasesPage.clickImportTestCaseButton()
+
+        //test case list table contains the group name of the test case that was imported
+        cy.get(TestCasesPage.testCaseListTable).should('contain.text', 'EncounterStatusMulti')
+        //test case list table contains the title of the test case that was imported
+        cy.get(TestCasesPage.testCaseListTable).should('contain.text', 'Person (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1)')
+
+    })
+    it('Verify error message when a Text file is imported', () => {
+        //navigate to the main measures page
+        cy.get(Header.measures).click()
+
+        //Click on Edit Button
+        MeasuresPage.clickEditforCreatedMeasure()
+
+        //Navigate to Test case list page
+        cy.get(EditMeasurePage.testCasesTab).click()
+
+        //click on the import test case button
+        cy.get(TestCasesPage.importTestCasesBtn).click()
+
+        //Attach Text file
+        cy.get(TestCasesPage.filAttachDropBox).attachFile(genericTextFile)
+
+        //error in import modal window
+        cy.get(TestCasesPage.testCaseImportErrorAtValidating).should('contain.text', 'An error occurred while validating the import file. Please try again or reach out to the Help Desk.')
+
+        //import button becomes unavailable
+        cy.get(TestCasesPage.importTestCaseModalBtn).should('be.disabled')
+    })
+
+    it('Verify error message when an invalid / empty Json file is imported', () => {
+        //navigate to the main measures page
+        cy.get(Header.measures).click()
+
+        //Click on Edit Button
+        MeasuresPage.clickEditforCreatedMeasure()
+
+        //Navigate to Test case list page
+        cy.get(EditMeasurePage.testCasesTab).click()
+
+        //click on the import test case button
+        cy.get(TestCasesPage.importTestCasesBtn).click()
+
+        //Upload invalid Json file
+        cy.get(TestCasesPage.filAttachDropBox).attachFile('example.json')
+
+        //message in import modal window about the file being empty
+        cy.get(TestCasesPage.testCaseImportErrorAtValidating).should('contain.text', 'No patients were found in the selected import file!')
+
+        //import button becomes unavailable
+        cy.get(TestCasesPage.importTestCaseModalBtn).should('be.disabled')
+    })
+
+    //Need to work with Ben to get the EICAR files allowed
+    it.skip('Verify error message when virus file is imported', () => {
```

Comment:
I think we can remove this, I don't think we will be able to test this at all

Reply:
This has been removed.